### DISCUSSION TITLE
超过1M的图片都上传失败

### DISCUSSION THREAD

**Discussion by yoke0104x** [lsky-org/lsky-pro#492] [问答]
大佬们你看我这个啥情况啊貌似超过1M的图片都上传失败 本地 和 文件存储 腾讯oss 阿里云 oss 都失败！

![image](https://user-images.githubusercontent.com/44898730/181423830-27b80a39-0d4c-41ed-802d-70225a11ea04.png)

**Comment by yoke0104x** [comment]
@gitter-badger @Menci @ctaoist @insoxin 大佬帮帮孩子

**Comment by 0xxb** [comment]
如果是 nginx，适量增加下图配置值：
![image](https://user-images.githubusercontent.com/22728201/181427233-5b478465-60e3-4e45-8ca5-6cd5c7806dd9.png)

**Reply by yoke0104x** [reply]
docker怎么搞呢？？？

**Reply by yoke0104x** [reply]
已解决多谢大佬

**Reply by loyalyonggang** [reply]
已经是图中的参数了，但还是无法上传成功

**Reply by jiehu0** [reply]
> 已解决多谢大佬

请问你是怎么解决的，我也是这样
